torchquad/utils/set_precision.py

```diff
@@ -26,8 +26,9 @@ def set_precision(data_type="float"):
             torch.set_default_tensor_type(torch.cuda.FloatTensor)
         else:
             logging.error(
-                f"{data_type} is not a valid data type. Only float or double are supported.
-                Setting default tensor type to cuda.Float32 (CUDA is initialized)."
+                data_type,
+                "is not a valid data type. Only float or double are supported. "
```

Comment:
Like this there is a white space missing. Will look like this:

`floatis not a valid data type. Only float or double are supported. `